midas2root/midas2root_mpmt.cxx

```diff
@@ -68,7 +68,10 @@
 
   // V1730 waveforms
   // 9.Nov.2017 Let's try adding using channels 2,3,4, 5
-  double V1730_wave0[nPoints_max][num_v1730_max],  V1730_wave1[nPoints_max][num_v1730_max],  V1730_wave2[nPoints_max][num_v1730_max], V1730_wave3[nPoints_max][num_v1730_max], V1730_wave4[nPoints_max][num_v1730_max], V1730_wave5[nPoints_max][num_v1730_max];
+  double V1730_wave0[nPoints_max][num_v1730_max],  V1730_wave1[nPoints_max][num_v1730_max],  V1730_wave2[nPoints_max][num_v1730_max], V1730_wave3[nPoints_max][num_v1730_max], V1730_wave4[nPoints_max][num_v1730_max], V1730_wave5[nPoints_max][num_v1730_max], V1730_wave6[nPoints_max][num_v1730_max], V1730_wave7[nPoints_max][num_v1730_max];
```

Comment:
Add another dimension for this array instead? (then can simplify branch creation loop below)

Reply:
Maybe.  I have some memory that there was a problem having too many levels of arrays in the TTree.  But we can try that.

Reply:
Can we leave that for a separate pull request though?